formats/redirect/publisher_v2/schema.json

```diff
@@ -118,7 +123,11 @@
           "enum": [ "prefix", "exact" ]
         },
         "destination": {
-          "$ref": "#/definitions/absolute_fullpath"
+          "type": "string",
+          "oneOf": [
```

Comment:
@robinjam @pmanrubia could we change this to `anyOf`?

Using `oneOf` means that the string will be invalid if it matches more than exactly one of the clauses.

In this case it's probably impossible for a destination to be both a valid `absolute_fullpath` AND `govuk_campaign_url`, but you could easily see a next developer adding something here that will make that possible. See https://github.com/alphagov/govuk-content-schemas/pull/397 for the same around specialist publisher documents.